### PR TITLE
Removes emag tells from airlocks

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

Removing emag tells from airlocks

### Why is this good for the game?

Improving RP

#### Changelog

:cl:  
rscdel: Removed emag tells from airlocks
/:cl:

### CODE REVIEW

**Comment by Tipy1802** [comment]
At leats security should be able to or you just made steal objectives brain dead

**Comment by Mqiib** [comment]
This just makes fixing emagged airlocks more of a pain in the ass than it already is.

**Comment by grimm-hollowknight** [comment]
Going to mini-maintainer here but

It is literally - what it removes - Improving RP.

How the hell does it improve RP? Do you think we're dumb enough just to lick this up because you jumped on a fad? Tell us why you think this is a good idea like we're not children.

I don't know what crevice in your mind thought that just putting this out here with no stock was a good idea, but the best thing you could probably do is take a break and put some actual effort in your PRs, rather than this pitiful shite.

Perhaps then you'd actually make some meaningful contributions.

**Comment by jcatjmeow** [comment]
Not super sold on the pr. Mind elaborating on your rationale?

**Comment by ghost** [comment]
@grimm-hollowknight 

hey

speaking as the resident cunt on github, can you stop being a cunt? there's no reason to be this belligerent right off the bat

**Comment by Marmio64** [comment]
how does this improve RP, ynot?

**Comment by Hopekz** [comment]
Without this "tell" you can just sit around and fuck with the airlock for who knows how long before you figure out it was emagged.
New players will find it really difficult as well as they have no prior experience to even know something like this is possible. They get a "hint" in the form of this text as to what the problem with the door is.

**Comment by MegaEmpirical** [comment]
The idea isn't bad at all, but I agree with the others about taking forever to realize it's fucked.

Should we make them fixable in return for them no longer having an easy tell? That was kind of the previous trade off.

**Comment by Darkstick** [comment]
"good for RP" is a terrible non-reason. What it should be is "removes the immediate meta "oh it's traitors" when finding a smoking door"

**Comment by maxion12345** [comment]
Not sure this is a good movie, I've always seen emags as the quick and dirty tool to access places, instead of taking actual effort to acquire tools. In exchange for your one-click access, you leave a tell. There is even a cheaper syndicate toolbox that comes with hacking tools for you to use if you can't be fucked to walk over to an autolathe.

Also i can already imagine the "I'm trying to fix this airlock but it won't close" mhelps already.

**Comment by JamieD1** [comment]
This was my idea..

Emag now instead of breaking electronics opens and bolts open the door, they can also emag it again to shut the door,

However there are sparks on the airlock to show its been damaged or messed with but this is no different from if someone wacked it a few times so noone would be able to know if its been "emagged" or just hit.

This would remove Emag meta and stop people bitching about them being hard to repair.

**Comment by alexkar598** [comment]
Tfw you buff the emag.

Why is this needed at all? What's wrong with a tell that theres a threat you need to find, get the detective to come over his tools and find the traitor scum.

**Comment by LoliconSlayer** [comment]
Ngl smells like an i ded PR

**Comment by ghost** [comment]
the idea i had if they wanted to make it improve RP was simply make the airlock show the emagged message on examine, but only to engineers, and a message that indicated the airlock was broken to security so that they couldn't quite tell the airlock was emagged but that it was nonfunctional 

then non-specialist crew wouldn't be able to tell precisely how an airlock they should have no idea works was broken, but people specializing in it would

**Comment by maxion12345** [comment]
It's my option that all traitor gear require a downside. In this case, your exchanging a quick and dirty door open for leaving a tell that there are traitors onboard. While adams idea for having just engineers be able to tell it's emagged, this also opens issues into questions like "I changed my job from assistant to Station Engineer, why can't I tell airlocks are emagged?" And "If there are no engineers, how can we figure out what's wrong with the door?" Or "I know this door is obviously emagged, but because there are no engineers I'm not allowed to say anything, this is pretty annoying". I also would think there are lore issues with this, because I'd like to believe Security would be trained to spot obvious tampering (smoking airlock, whatever), and would at least be able to know something is up. Why would an engineer be able to go "aw yup dats emagged" but a security officer...wouldn't?

Tl:dr
Emag buff is emag buff
Not sure what this will do for rp, especially because people will end up figuring out it's emagged based off of its behavior. You could claim this would be metagaming, but emags aren't stealthy items regardless. Also, with how easy it would be to determine, it'd end up having security on edge for trouble even if they wouldn't be able to go "emagged door". Also your lack of any response other then "improves rp" feels like to me your not sold on the idea either

**Comment by alexkar598** [comment]
I'm not entirely convinced this isnt a meme PR, it looks like someone made a PR with improves rp as the sole reason and they're trying to see if they can get it merged despite not giving any meaningful reason

**Comment by maxion12345** [comment]
> I'm not entirely convinced this isnt a meme PR, it looks like someone made a PR with improves rp as the sole reason and they're trying to see if they can get it merged despite not giving any meaningful reason

But alecks car! We're mrp and this pr /clearly/ states it improves rp. Ignore the fact it adds my ckey to the council list, and merge it! Or are you anti rp?????

**Comment by ynot01** [comment]
i forgot this pr existed sorry ill uh try to do something with it

**Comment by alexkar598** [comment]
Every single fucking server out there has tried to nerf the emag or at the very least not buff it.

Buffing the emag is silly, especially its let me in here ability, especially talking about removing its only counter(bolting airlocks) and ESPECIALLY in order to "enhance roleplay" whatever that means in this case.

**Comment by ynot01** [comment]
another project lost to time and lack of motivation
